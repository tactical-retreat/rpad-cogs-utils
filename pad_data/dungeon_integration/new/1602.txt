Gran Reverse Descended!-Dragon Enhanced - Inverted Eternal Dragon-Mythical

stage 0

1093 - Tornado Holy Dragon @ level 10
Fertile Guard
Voids status ailments for 999 turns
----- Execute below actions in order repeatedly -----
ESCountdown
Display <COUNTER> and skip turn
ESCountdown
Display <COUNTER> and skip turn
ESCountdown
Display <COUNTER> and skip turn
Disaster Air
140580 Damage - Deal 300% damage (3 hits, 100% each)

1094 - Cyclone Devil Dragon @ level 5
Demonic Stare
Bind Light cards for 4 turns
----- Turn 1 -----
Hell Ball
18587 Damage - Change Water to Dark, Deal 90% damage (1 time use)
----- HP <= 49 -----
Dark Cyclone Blast
25815 Damage - Blind all orbs on the board, Deal 125% damage

stage 1

971 - Dark Wood Skydragon, Horai @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Wood Sky
5783 Damage - Deal 120% damage (2 hits, 60% each)
----- HP <= 50 -----
Mistral Breath
7229 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 150% damage

1132 - Dark Red Skydragon, El Dorado @ level 5
Dark Skydragon Flight
4179 Damage - Deal 85% damage
----- Execute below actions in order repeatedly -----
ESCountdown
Display <COUNTER> and skip turn
ESCountdown
Display <COUNTER> and skip turn
ESCountdown
Display <COUNTER> and skip turn
Hellfire Breath
73749 Damage - Deal 1500% damage (3 hits, 500% each)

1133 - Dark Blue Skydragon, Nirai Kanai @ level 5
Dark Skydragon Flight
4002 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Blue Sky
5650 Damage - Deal 80%~120% damage (2~3 hits, 40% each)
----- HP <= 50 -----
Exa-Freeze Breath
7062 Damage - Change 4th, 5th row to Water orbs, Deal 150% damage

1134 - Dark Holy Skydragon, Shangri-La @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Holy Sky
5783 Damage - Deal 100%~120% damage (5~6 hits, 20% each)
----- HP <= 50 -----
Gigavolt Breath
7229 Damage - Bind 6 random for 1 turns, Deal 150% damage (1 time use)

1135 - Dark Night Skydragon, Elysium @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Night Sky
6024 Damage - Deal 100%~125% damage (4~5 hits, 25% each)
----- HP <= 50 -----
Deep Shadow Breath
7229 Damage - Blind all orbs on the board, Deal 150% damage

stage 2

1645 - Marvelous Red Dragon Caller, Sonia @ level 5
You will go no further!
Absorb combo <= 5 damage for 99 turns
----- HP <= 100 -----
Magic Dragon Sword - Shooting Star
16402 Damage - Change Random to Fire, Deal 100% damage (70% chance)
----- HP <= 75 -----
Magic Dragon Sword - Lightning
8201 Damage - Bind 1 random for 2 turns, Deal 50% damage (50% chance) (1 time use)
----- HP <= 50 -----
Magic Dragon Sword - Chaos
12302 Damage - Blind all orbs on the board, Deal 75% damage (40% chance)
----- HP <= 30 -----
Draco Summon-Twin Dragon King
Do nothing
Flame Emperor
82013 Damage - Change Random to Fire, Deal 500% damage

stage 3

2567 - Past Life Jade Dragon Caller, Sonia @ level 5
You're not getting through!
Player -99% HP
----- Turn 1 -----
Gale Shot
16666 Damage - Change 3rd column to Wood orbs, Deal 150% damage (1 time use)
----- HP <= 100 -----
Poison Arrow
12222 Damage - Spawn random 2 Poison orbs, Deal 110% damage
----- HP <= 69 -----
Ancient Dragon Tribe's Dark Curse
Bind active skills for 6 turns
----- HP <= 29 -----
Draco Summon-Swift Dragon Emperor
Do nothing
----- HP <= 29 -----
World-Wrecking Windstorm
277777 Damage - Deal 2500% damage (5 hits, 500% each)

stage 4

2566 - Ancestral Blue Dragon Caller, Sonia @ level 5
I won't let you go
Lock all Fire, Water, Wood, Light, Dark, Heal, Jammer, Poison, Mortal Poison orbs
----- Turn 1 -----
Aqua Sphere
12711 Damage - Change Random to Water, Deal 130% damage (40% chance)
Cold Scythe
10755 Damage - Delay active skills by 1 turns, Deal 110% damage (40% chance)
Crystallizing Dark Wave
8800 Damage - Bind 2 random for 3 turns, Deal 90% damage (20% chance) (1 time use)
----- HP <= 100 -----
Aqua Sphere
12711 Damage - Change Random to Water, Deal 130% damage (40% chance)
Cold Scythe
10755 Damage - Delay active skills by 1 turns, Deal 110% damage (40% chance)
----- HP <= 29 -----
Draco Summon-Frost Dragon Emperor
Do nothing
----- HP <= 29 -----
Abyss Tempest
48888 Damage - Deal 500% damage (5 hits, 100% each)

stage 5

2737 - Dusking Dragon Caller, Sonia Gran Reverse @ level 5
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Fire, Light, Dark damage for 3 turns (33% chance)
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Water, Light, Dark damage for 3 turns (34% chance)
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Wood, Light, Dark damage for 3 turns (33% chance)
----- Turn 1 -----
Draco Summon-Dragon Portal
Do nothing (1 time use)
----- Turn 1 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 2 -----
Cattle Mutilation
16600 Damage - Blind all orbs on the board, Deal 160% damage (20% chance)
Glass Leaf
15562 Damage - Lock all Wood orbs, Deal 150% damage (20% chance)
Genocide Missile Type 0
12450 Damage - Change Random 6 to Poison, Deal 120% damage (20% chance)
Dark Rumble
15562 Damage - Change Random to Dark, Deal 150% damage (20% chance)
Brilliant Dragon Skull
14525 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 140% damage (20% chance)
----- Turn 2 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 3 -----
Fair Skies Guardian Dragon - Shield
Reduce damage from all sources by 50% for 2 turns (1 time use)
----- Turn 3 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 4 -----
Cattle Mutilation
16600 Damage - Blind all orbs on the board, Deal 160% damage (20% chance)
Glass Leaf
15562 Damage - Lock all Wood orbs, Deal 150% damage (20% chance)
Genocide Missile Type 0
12450 Damage - Change Random 6 to Poison, Deal 120% damage (20% chance)
Dark Rumble
15562 Damage - Change Random to Dark, Deal 150% damage (20% chance)
Brilliant Dragon Skull
14525 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 140% damage (20% chance)
----- Turn 4 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 5 -----
Fair Skies Guardian Dragon - Skill
Absorb combo <= 6 damage for 2 turns (1 time use)
----- Turn 5 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 6 -----
Cattle Mutilation
16600 Damage - Blind all orbs on the board, Deal 160% damage (20% chance)
Glass Leaf
15562 Damage - Lock all Wood orbs, Deal 150% damage (20% chance)
Genocide Missile Type 0
12450 Damage - Change Random 6 to Poison, Deal 120% damage (20% chance)
Dark Rumble
15562 Damage - Change Random to Dark, Deal 150% damage (20% chance)
Brilliant Dragon Skull
14525 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 140% damage (20% chance)
----- Turn 6 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 7 -----
Fair Skies Guardian Dragon - Balance
Void damage >= 800000 for 2 turns (1 time use)
----- Turn 7 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 8 -----
Cattle Mutilation
16600 Damage - Blind all orbs on the board, Deal 160% damage (20% chance)
Glass Leaf
15562 Damage - Lock all Wood orbs, Deal 150% damage (20% chance)
Genocide Missile Type 0
12450 Damage - Change Random 6 to Poison, Deal 120% damage (20% chance)
Dark Rumble
15562 Damage - Change Random to Dark, Deal 150% damage (20% chance)
Brilliant Dragon Skull
14525 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 140% damage (20% chance)
----- Turn 8 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 9 -----
Fair Skies Guardian Dragon - Judge
Absorb damage >= 1500000 damage for 2 turns (1 time use)
----- Turn 9 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 10 -----
Cattle Mutilation
16600 Damage - Blind all orbs on the board, Deal 160% damage (20% chance)
Glass Leaf
15562 Damage - Lock all Wood orbs, Deal 150% damage (20% chance)
Genocide Missile Type 0
12450 Damage - Change Random 6 to Poison, Deal 120% damage (20% chance)
Dark Rumble
15562 Damage - Change Random to Dark, Deal 150% damage (20% chance)
Brilliant Dragon Skull
14525 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 140% damage (20% chance)
----- Turn 10 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- Turn 11 -----
Draggie Knight, Zeal's Enhancement + Draggie Shield
Increase damage to 250% for the next 1 turns + Reduce damage from all sources by 75% for 1 turns (1 time use)
----- Turn 11 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)
----- HP <= 100 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)

