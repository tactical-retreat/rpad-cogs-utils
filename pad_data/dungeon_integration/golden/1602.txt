Gran Reverse Descended!-Dragon Enhanced - Inverted Eternal Dragon-Mythical

stage 0

1093 - Tornado Holy Dragon @ level 10
Basic attacks (1)

1094 - Cyclone Devil Dragon @ level 5
Demonic Stare
Bind Light cards for 4 turns
----- Turn 1 -----
Hell Ball
18587 Damage - Change Water to Dark, Deal 90% damage
----- HP <= 49 -----
Dark Cyclone Blast
25815 Damage - Blind all orbs on the board, Deal 125% damage

stage 1

971 - Dark Wood Skydragon, Horai @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Wood Sky
5783 Damage - Deal 120% damage (2 hits, 60% each)
----- HP <= 50 -----
Mistral Breath
7229 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Deal 150% damage

1132 - Dark Red Skydragon, El Dorado @ level 5
Basic attacks (1)

1133 - Dark Blue Skydragon, Nirai Kanai @ level 5
Dark Skydragon Flight
4002 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Blue Sky
5650 Damage - Deal 80%~120% damage (2~3 hits, 40% each)
----- HP <= 50 -----
Exa-Freeze Breath
7062 Damage - Change 4th, 5th row to Water orbs, Deal 150% damage

1134 - Dark Holy Skydragon, Shangri-La @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Holy Sky
5783 Damage - Deal 100%~120% damage (5~6 hits, 20% each)
----- HP <= 50 -----
Gigavolt Breath
7229 Damage - Bind 6 random for 1 turns, Deal 150% damage

1135 - Dark Night Skydragon, Elysium @ level 5
Dark Skydragon Flight
4096 Damage - Deal 85% damage
----- HP <= 100 -----
Breath of the Night Sky
6024 Damage - Deal 100%~125% damage (4~5 hits, 25% each)
----- HP <= 50 -----
Deep Shadow Breath
7229 Damage - Blind all orbs on the board, Deal 150% damage

stage 2

1645 - Marvelous Red Dragon Caller, Sonia @ level 5
You will go no further!
Absorb combo <= 5 damage for 99 turns
----- HP <= 100 -----
Magic Dragon Sword - Shooting Star
16402 Damage - Change Random to Fire, Deal 100% damage
----- HP <= 49 -----
Magic Dragon Sword - Shooting Star
16402 Damage - Change Random to Fire, Deal 100% damage
----- HP <= 75 -----
Magic Dragon Sword - Lightning
8201 Damage - Bind 1 random for 2 turns, Deal 50% damage
----- HP <= 50 -----
Magic Dragon Sword - Chaos
12302 Damage - Blind all orbs on the board, Deal 75% damage
----- HP <= 30 -----
Draco Summon-Twin Dragon King
Do nothing
Flame Emperor
82013 Damage - Change Random to Fire, Deal 500% damage
----- HP <= 30 -----
Flame Emperor
82013 Damage - Change Random to Fire, Deal 500% damage

stage 3

2567 - Past Life Jade Dragon Caller, Sonia @ level 5
You're not getting through!
Player -99% HP
----- Turn 1 -----
Gale Shot
16666 Damage - Change 3rd column to Wood orbs, Deal 150% damage
----- HP <= 100 -----
Poison Arrow
12222 Damage - Spawn random 2 Poison orbs, Deal 110% damage
----- HP <= 69 -----
Ancient Dragon Tribe's Dark Curse
Bind active skills for 6 turns
----- HP <= 29 -----
Draco Summon-Swift Dragon Emperor
Do nothing
----- HP <= 29 -----
World-Wrecking Windstorm
277777 Damage - Deal 2500% damage (5 hits, 500% each)

stage 4

2566 - Ancestral Blue Dragon Caller, Sonia @ level 5
I won't let you go
Lock all Fire, Water, Wood, Light, Dark, Heal, Jammer, Poison, Mortal Poison orbs
----- Turn 1 -----
Aqua Sphere
12711 Damage - Change Random to Water, Deal 130% damage
Cold Scythe
10755 Damage - Delay active skills by 1 turns, Deal 110% damage
Crystallizing Dark Wave
8800 Damage - Bind 2 random for 3 turns, Deal 90% damage
----- HP <= 100 -----
Aqua Sphere
12711 Damage - Change Random to Water, Deal 130% damage
Cold Scythe
10755 Damage - Delay active skills by 1 turns, Deal 110% damage
----- HP <= 29 -----
Draco Summon-Frost Dragon Emperor
Do nothing
----- HP <= 29 -----
Abyss Tempest
48888 Damage - Deal 500% damage (5 hits, 100% each)

stage 5

2737 - Dusking Dragon Caller, Sonia Gran Reverse @ level 5
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Fire, Light, Dark damage for 3 turns
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Water, Light, Dark damage for 3 turns
Let's have some fun + Dragon's Origin
Voids status ailments for 999 turns + Absorb Wood, Light, Dark damage for 3 turns
----- HP <= 100 -----
Alt. Five Origin Dragon Star Attack
51875 Damage - Deal 500% damage (5 hits, 100% each)

